### PR TITLE
fix handling of exitCodes during SCP

### PR DESCRIPTION
**Pull request by alex-kovoy**
when calling teleport scp on the remote host, make sure that we are passing the exitCode error when shell command fails to execute.

### CODE REVIEW

File: lib/sshutils/scp/scp.go [hunk 1]
```diff
@@ -176,7 +176,7 @@ func CreateCommand(cfg Config) (Command, error) {
 			"LocalAddr":      cfg.Flags.LocalAddr,
 			"RemoteAddr":     cfg.Flags.RemoteAddr,
 			"Target":         cfg.Flags.Target,
-			"User":           cfg.User,
```
Comment (klizhentas) [inline lib/sshutils/scp/scp.go]: why did you remove `User` line here?

**Review by klizhentas — APPROVED** [review]

**Review by russjones — APPROVED** [review]
